apps/server/src/modules/board/service/drawing-do-authorizable.service.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { BoardExternalReferenceType, BoardRoles } from '@shared/domain/domainobject';
+import { CourseRepo } from '@shared/repo';
+import { cardFactory, courseFactory, roleFactory, setupEntities, userFactory } from '@shared/testing';
+import { columnBoardFactory, columnFactory } from '@shared/testing/factory/domainobject';
+import { DrawingDoRepo } from '../repo';
+import { DrawingDoAuthorizableService } from '.';
+
+describe(DrawingDoAuthorizableService.name, () => {
+	let module: TestingModule;
+	let service: DrawingDoAuthorizableService;
+	let drawingDoRepo: DeepMocked<DrawingDoRepo>;
+	let courseRepo: DeepMocked<CourseRepo>;
+
+	beforeAll(async () => {
+		module = await Test.createTestingModule({
+			providers: [
+				DrawingDoAuthorizableService,
+				{
+					provide: DrawingDoRepo,
+					useValue: createMock<DrawingDoRepo>(),
+				},
+				{
+					provide: CourseRepo,
+					useValue: createMock<CourseRepo>(),
+				},
+			],
+		}).compile();
+
+		service = module.get(DrawingDoAuthorizableService);
+		drawingDoRepo = module.get(DrawingDoRepo);
+		courseRepo = module.get(CourseRepo);
+		await setupEntities();
+	});
+
+	afterAll(async () => {
+		await module.close();
+	});
+
+	describe('findById', () => {
+		describe('when finding a board domainobject', () => {
+			const setup = () => {
+				const columnBoard = columnBoardFactory.build();
+				drawingDoRepo.findById.mockResolvedValue(columnBoard);
+
+				return { columnBoardId: columnBoard.id };
+			};
+
+			it('should call the repository', async () => {
+				const { columnBoardId } = setup();
+
+				await service.findById(columnBoardId);
+
+				expect(drawingDoRepo.findById).toHaveBeenCalledWith(columnBoardId, 1);
+			});
+
+			it('should return the column', async () => {
+				const { columnBoardId } = setup();
+
+				const result = await service.findById(columnBoardId);
+
+				expect(result.id).toEqual(columnBoardId);
+			});
+		});
+	});
+
+	describe('getBoardAuthorizable', () => {
+		describe('when having an empty board', () => {
```

Comment:
this is measleading, don't know what you try to do actually. 
yes, you have an empty board (no collumns, cards, elements, etc), but actually you have no course and therefor no users associated.